vspec2x.py

```diff
@@ -31,6 +31,7 @@ def export(config: argparse.Namespace, root: VSSNode):
     json = vss2json
     csv = vss2csv
     yaml = vss2yaml
+    binary = vss2binary
```

Comment:
Is it this that gives build problem - that we now need the generated binary also when generating e.g. csv or json?

(Which we likely do not want)

Reply:
Disclaimer: Have not tested it, but it might be possible to move this assignments in vss2binary.py

```python
dllName = "../binary/binarytool.so"
 dllAbsPath = os.path.dirname(os.path.abspath(__file__)) + os.path.sep + dllName
 _cbinary = ctypes.CDLL(dllAbsPath)
```

inside the call to export()?

In that case it would only explode IF you want to convert 2binary and forgot to compile. (which seems fine)